.github/workflows/build.yaml

```diff
@@ -68,3 +68,31 @@ jobs:
           RAILS_ENV: test
           RSPEC_RETRY_RETRY_COUNT: 3
         run: bundle exec rake
+  migrations:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgis/postgis:11-2.5
+        env:
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: weg-li_test
+        ports: ["5432:5432"]
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          cache: yarn
+          node-version: 18
```

Comment:
Same as above goes here. Make the whole manual caching magic above obsolete. Also I would really propose to use `bun` instead of `yarn`. It is even way more fun than yarn and soooo fast.

Reply:
I will do this later in another PR. Probably not use bun for the time being, I will see then. Out of scope of this PR anyhow.